<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/K_logo_small.png</normaloff>:/image/K_logo_small.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>79</y>
      <width>791</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>671</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="LedControl" name="widget_channel1" native="true"/>
       </item>
       <item>
        <widget class="LedControl" name="widget_channel2" native="true"/>
       </item>
       <item>
        <widget class="LedControl" name="widget_channel3" native="true"/>
       </item>
       <item>
        <widget class="LedControl" name="widget_channel4" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="widget_pulse" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>771</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="PulseMode" name="widget_pulsechannelAll" native="true"/>
       </item>
       <item>
        <widget class="PulseMode" name="widget_pulsechannel1" native="true"/>
       </item>
       <item>
        <widget class="PulseMode" name="widget_pulsechannel2" native="true"/>
       </item>
       <item>
        <widget class="PulseMode" name="widget_pulsechannel3" native="true"/>
       </item>
       <item>
        <widget class="PulseMode" name="widget_pulsechannel4" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="toolButton_connect">
       <property name="text">
        <string>连接</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/connect.png</normaloff>:/image/connect.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_connect">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_mode">
       <property name="text">
        <string>模式</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/main_config.png</normaloff>:/image/main_config.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_mode">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_language">
       <property name="text">
        <string>语言</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/language.png</normaloff>:/image/language.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_help">
       <property name="text">
        <string>帮助</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/help.png</normaloff>:/image/help.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LedControl</class>
   <extends>QWidget</extends>
   <header>ledcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PulseMode</class>
   <extends>QWidget</extends>
   <header>pulsemode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
